### PR TITLE
Release/1.3.0.1

### PR DESCRIPTION
**Pull request by h3xar0n**

### CODE REVIEW

File: app/_data/docs_nav_ee_1.3-x.yml [hunk 1]
```diff
@@ -139,6 +141,8 @@
         url: /admin-api/vitals/vitals-prometheus-strategy
   - text: Audit Logging
     url: /admin-api/audit-log
+  - text: Keyring and Data Encryption
```
Comment (levfishbluefish) [inline app/_data/docs_nav_ee_1.3-x.yml]: Looks like we are linking this in the nav twice? was this on purpose?

Comment (h3xar0n) [reply app/_data/docs_nav_ee_1.3-x.yml]: @p0pr0ck5 , these seem to point to different paths.

Comment (h3xar0n) [reply app/_data/docs_nav_ee_1.3-x.yml]: I see now. One is an overview, the other is API reference.

File: app/enterprise/changelog.md [hunk 2]
```diff
@@ -3,6 +3,89 @@ title: Kong Enterprise Changelog
...
 ---
 
+## 1.3.0.1
+**Release Date:** 2019/12/19
+
+### Features
+
+* Kong Enterprise now officially supports RHEL 8
+* (Alpha feature) Kong Enterprise can now perform encryption-at-rest for sensitive fields within the data store (Postgres or Cassandra).
+* (Alpha feature) Kong Enterprise now has a License Reports module for customers to view current usage metrics. For more information, contact your Kong Account Executive.
```
Comment (levfishbluefish) [inline app/enterprise/changelog.md]:
```suggestion
```

Comment (h3xar0n) [reply app/enterprise/changelog.md]: This needs to be in both 1.3 and 1.3.0.1, correct, @GayleNeumann ?

**Pull request merged**